deploy/charts/harvester/values.yaml

```diff
@@ -341,13 +341,13 @@ jobs:
 harvester-network-controller:
   ## Specify to install harvester network controller,
   ## defaults to "false".
-  enabled: false
+  enabled: true
 
   image:
     repository: rancher/harvester-network-controller
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: master-head
+    tag: v0.2.5
```

Comment:
those changes are not needed as we will tag the version on the tag branch, but I think the helper image values are missing.